postgres/main.tf

```diff
@@ -75,15 +75,6 @@ resource "aws_security_group" "default" {
     self = true
   }
 
-  egress {
-    description = "Egress for HTTPS"
-    from_port   = 443
-    to_port     = 443
-    protocol    = "tcp"
-
-    cidr_blocks = ["0.0.0.0/0"]
-  }
```

Comment:
Interesting that this did not result in a failing test. Have you run the end to end tests?

Reply:
I was a bit concerned by that too. I ran `terraform test` from the postgres module. Is there anything else I need to run? I can try and add one for the other egress rules, so that we can have more confidence in future

Reply:
I think I'm going to spend a little time and refactor to use rule resources instead of inline egress and ingress blocks. It's recommended and will make writing tests easier. As per this note https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group

Reply:
I just looked and there are no end to end tests for this module. Hopefully that will change during the course of working through [this epic](https://uktrade.atlassian.net/browse/DBTP-980).